rebasehelper/utils.py

```diff
@@ -101,7 +101,15 @@ class ConsoleHelper(object):
 
     @classmethod
     def should_use_colors(cls, conf):
-        """Determine whether ansi colors should be used for CLI output"""
+        """Determine whether ansi colors should be used for CLI output
```

Comment:
This shouldn't be in imperative mode. Also, ANSI is an abbreviation, so it should be capitalized.

Reply:
And it should end with a `.`.